handlers/handler-loadbalance/src/main/java/org/apache/servicecomb/loadbalance/Configuration.java

```diff
@@ -173,7 +173,7 @@ public int getRetryOnSame(String microservice) {
   }
 
   public boolean isIsolationFilterOpen(String microservice) {
-    String p = getStringProperty("false",
+    String p = getStringProperty("true",
```

Comment:
There are lots of default value changed, not sure if it break the old behavior.

Reply:
Isolation is not enabled by old version, now we enable it by default and change it's default values. Documents here: https://huaweicse.github.io/servicecomb-java-chassis-doc/zh_CN/references-handlers/loadbalance.html